omero/changelog.rst

```diff
@@ -0,0 +1,25 @@
+CHANGELOGS
+==========
+
+
+Links to decoupled 121323repositories
```

Comment:
Where does the 121323 come from?

Reply:
oops. This should not be there. I was testing text